src/sass/atoms/_layout.scss

```diff
@@ -1,69 +1,10 @@
-.t-0 {
-    top: 0;
-}
-
-.b-0 {
-    bottom: 0;
-}
-
-.l-0 {
-    left: 0;
-}
-
-.r-0 {
-    right: 0;
-}
-
-.hidden {
-    display: none;
-}
-
-.inline {
-    display: inline;
-}
-
-.block {
-    display: block;
-}
-
-.inline-block {
-    display: inline-block;
-}
-
-.absolute {
-    position: absolute;
-}
-
-.relative {
-    position: relative;
-}
-
-.o-hidden {
-    overflow: hidden;
-}
-
-.o-y-auto {
-    overflow-y: auto;
-}
-
-@include reactiveClasses('left') {
-    float: left;
-}
-
-@include reactiveClasses('right') {
-    float: right;
-}
-
-.full-width {
-    width: 100%;
-}
-
-.full-height {
-    height: 100%;
-}
-
-.fixed {
-    position: fixed;
+/**
+    Some of the Once's classes are created using the mapping in _classes.scss
+*/
+@each $classKey, $className in $classes {
```

Comment:
Part of creating css style is actually done in _functions.scss. It can be used, and to add some array love for it that each bock could be moved into some @mixin createSelectors (again a bit poor naming..)

```
// _functions.scss, line 66
@mixin createSelector($selectorName, $properties) // can be used
```